eamena/eamena/models/forms.py

```diff
@@ -114,6 +114,45 @@ def load(self, lang):
                     'GENERAL_DESCRIPTION_TYPE.E55' : Concept().get_e55_domain('GENERAL_DESCRIPTION_TYPE.E55')
                 }
             }
+            
+class FeatureSummaryForm(ResourceForm):
+    @staticmethod
+    def get_info():
+        return {
+            'id': 'summary-feature',
+            'icon': 'fa-align-center',
+            'name': _('Resource Summary'),
```

Comment:
Resource Summary should be called 'Feature Summary' instead at the Feature level.